Control_Toolkit_ASF_Template/config_optimizers.yml

```diff
@@ -12,7 +12,6 @@ cem-tf:
   cem_best_k: 40
   warmup: false
   warmup_iterations: 250
-  predictor_specification: "ODE_TF"    # Can be "ODE", "ODE_TF", network/GP name (possibly with path) e.g. 'GRU-6IN-32H1-32H2-5OUT-0'/'SGP_30' or a name of a custom predictor. For more info see config_predictors in SI_Toolkit_ASF
```

Comment:
Removed predictor_specification from optimizer config.

Predictor choice should be with the controller config only.